Cargo.toml

```diff
@@ -20,6 +20,7 @@ default = ["clock", "std", "wasmbind"]
 alloc = []
 libc = []
 std = []
+serde = ["dep:serde", "dep:serde_json"]
```

Comment:
This will be causing the MSRV failure (one of the two CI failures) due to:

> Note: The dep: syntax is only available starting with Rust 1.60. Previous versions can only use the implicit feature name.

There are suggestions that we will probably go higher than the current `1.48` (see: https://github.com/chronotope/chrono/issues/995), but it may not go as high as `1.60` (put another way, we probably can't justify moving to `1.60` just to get this `dep:{crate}` syntax)